apps/morphologyConverter.cpp

```diff
@@ -3,140 +3,141 @@
  *                          Stefan.Eilemann@epfl.ch
  */
 
+#include <boost/foreach.hpp>
+#include <boost/program_options.hpp>
 #include <brion/brion.h>
 #include <lunchbox/clock.h>
 #include <lunchbox/file.h>
 #include <lunchbox/term.h>
-#include <boost/program_options.hpp>
-#include <boost/foreach.hpp>
```

Comment:
The old include order was correct. Add a newline so clang-format does not move it.